enclave_build/docs/cache.md

```diff
@@ -0,0 +1,127 @@
+# Image Cache
+
+When building an enclave image using the Docker daemon, the image will be pulled and stored in the Docker cache. We want to implement the same functionality when choosing to build an OCI image. The resulted cache will follow the [OCI specification](https://github.com/opencontainers/image-spec), allowing tools like `linuxkit` to use it as its own cache, and also adding images from OCI archive with the same format.  
+
+## Structure
+
+The cache structure is OCI compliant and follows the specification:
+https://github.com/opencontainers/image-spec/blob/main/spec.md
+
+The root folder of the cache is either `${XDG_DATA_HOME}/.aws-nitro-enclaves-cli/container_cache`, or, if `XDG_DATA_HOME` is not set, `${HOME}/.local/share/.aws-nitro-enclaves-cli/container_cache` is used, as in the [specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).
+
+The cache holds common components used by all images (index file, layout file, blobs folder) and individual items specific to the image named blobs (manifest, config file, layers). A possible cache structure could look like this:
+
+```
+cache_dir/
+|________index.json        -> will point to the manifest blob for each image
+|________oci-layout
+|            blobs/sha256/
+|            |_______98806831de537d56f19b4b3cbf6ed80187374a1521a18d2d0c5f8a0e3962b2ae -> manifest_hash will point to config and layers
+|            |_______feb5d9fea6a5e9606aa995e879d862b825965ba48de054caab5ef356dc6b3412 -> config_hash
+|            |_______2db29710123e3e53a794f2694094b9b4338aa9ee5c40b930cb8063a1be392c54 -> layer_hash
+```
+
+### Examples
+
+#### Index
+
+index.json
+```
+{
+  "schemaVersion": 2,
+  "manifests": [
+    {
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+      "size": 425,
+      "digest": "sha256:98806831de537d56f19b4b3cbf6ed80187374a1521a18d2d0c5f8a0e3962b2ae",
+      "annotations": {
+        "org.opencontainers.image.ref.name": "docker.io/test-oci:hello"
+      }
+    }
+  ]
+}
+```
+
+As we can see, the index holds one manifest entry, for one image stored. The number of manifests increases with each newly stored image. The manifest hash will point us to the manifest blob.
```

Comment:
As already mentioned, please avoid the use of `will`. Makes the doc hard to read.